interview_db_vue/components/student/interview-listing.vue

```diff
@@ -4,6 +4,7 @@
   <button
     type="button"
     class="btn-card mt-5"
+    style="height=21rem"
```

Comment:
syntax should be `style="height:21rem"`

Is this intended to be a HUGE button?

Reply:
Yeah, they're clickable cards, so I wrapped it in a button to make the entire card clickable--is there a better way to do that?